prickly-pufferfish/python_questions/add_to_zero.py

```diff
@@ -0,0 +1,16 @@
+"""
+Write a function with a list of ints as a paramter. /
+Return True if any two nums sum to 0. /
+
```

Comment:
Are the questions repeated more than once?  There seems to be 3 separate places that have the questions listed or written out.